projects/smoke_drv120/src/main.c

```diff
@@ -0,0 +1,57 @@
+#include <stdbool.h>
+
+#include "delay.h"
+#include "drv120_relay.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+// Toggle the following to select which data is logged
+#define ENABLE_PIN \
+  { .port = GPIO_PORT_A, .pin = 8 }
+#define STATUS_PIN \
+  { .port = GPIO_PORT_A, .pin = 6 }
+#define TIME_INTERVAL 500
```

Comment:
500ms is *really* fast if we want to observe anything, I'd set it to 1000ms or 2000ms.